assets/src/edit-story/components/inspector/document/publish/index.js

```diff
@@ -157,20 +178,19 @@ function PublishPanel() {
   );
 
   const authorLabel = __('Author', 'web-stories');
-  const use12HourFormat = is12Hour(timeFormat);
   return (
     <SimplePanel name="publishing" title={__('Publishing', 'web-stories')}>
       <Row>
         <FieldLabel>{__('Publish', 'web_stories')}</FieldLabel>
```

Comment:
I think clicking this label should also show the date picker. In general our labels should focus the input fields and/or simulate when happens when clicking the corresponding input field.